desktop_version/src/titlerender.cpp

```diff
@@ -530,6 +530,12 @@ void titlerender(Graphics& dwgfx, mapclass& map, Game& game, entityclass& obj, U
                 {
                     dwgfx.Print( -1, 105, "Game speed is at 40%", tr, tg, tb, true);
                 }
+            } else if (game.currentmenuoption == 4) {
+                dwgfx.bigprint(-1, 30, "Fake load screen", tr, tg, tb, true);
```

Comment:
Brace on a separate line from else if, also capitalization on Fake Load Screen. Otherwise looks good.

Reply:
Fixed in dd2cdaddf7a0202c07017e92cbbd01c80029b638.